routes/events.js

```diff
@@ -194,41 +204,32 @@ function promisifyInputs(body, geoinfo, supporter) {
     /* this big debug noise is handy on the server */
     if(processed.timelines && processed.timelines[0] && processed.timelines[0].nonfeed)
         debug(" * non-newsfeed navigation: no content received");
-    else{
-	processed_timelines = _.size(processed.timelines)
-	processed_impressions =  _.size(processed.impressions)
-	processed_html = _.size(processed.htmls)
-    	
+    else {
+        processed_timelines = _.size(processed.timelines)
+        processed_impressions =  _.size(processed.impressions)
+        processed_html = _.size(processed.htmls)
+        
         debug(" * %d timelines %d impressions %d html %s",
-	      processed_timelines,
-	      processed_impressions,
-	      processed_html,
+            processed_timelines, processed_impressions, processed_html,
             _.get(processed.timelines[0], 'tagId') ?
                 "tagId " + processed.timelines[0].tagId :
                 ""
             );
     }
-    
-    debug(" * user %d [%s] last activity %s (%s ago) %s",
-        supporter.userId, geoinfo,
-        moment(supporter.lastActivity).format("HH:mm DD/MM"),
-        moment.duration(moment.utc()-moment(supporter.lastActivity)).humanize(),
-          supporter.version);
-    
+
     var ts = Math.round((new Date()).getTime() / 1000);
```

Comment:
@vecna let's think if we should put a milliseconds version or a random UUID version here